docs/conf.py

```diff
@@ -193,3 +194,53 @@
 # Current scipy hosted docs are missing the object.inv file so leaving this
 # commented out until the missing file is added back.
 #                       'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None)}
+
+# Prepend warning for development docs:
+
+if not os.getenv("EXPERIMENTS_DEV_DOCS", None):
+    rst_prolog = """
+.. raw:: html
+    <br><br><br>
+""".format(
+        release
+    )
+else:
+    rst_prolog = """
+.. raw:: html
+    <br><br><br>
+.. note::
+    This is the documnetation for the current state of the development branch
+    of Qiskit Experiments. The documentation or APIs here can change prior to being
+    released.
+"""
+
+
+def _get_versions(app, config):
+    context = config.html_context
+    start_version = (0, 1, 0)
```

Comment:
This will result in a bunch of dead links in the menu. This is basically saying that there are stable versions of the docs available at `./stable/*` for every minor version tag from 0.1.0 to the current release. The only way to avoid this is to manually build and upload the versions. I can do this locally for you, but maybe it's only worth doing this from the current release moving forward.